x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/view/host_isolation_exceptions_list.tsx

```diff
@@ -55,10 +55,16 @@ export const HostIsolationExceptionsList = () => {
 
   const [itemToDelete, setItemToDelete] = useState<ExceptionListItemSchema | null>(null);
 
+  const includedPoliciesParam = location.included_policies;
+
   const { isLoading, data, error, refetch } = useFetchHostIsolationExceptionsList({
     filter: location.filter,
     page: location.page_index,
     perPage: location.page_size,
+    policies:
+      includedPoliciesParam && includedPoliciesParam !== ''
```

Comment:
The first check is redundant no? `includedPoliciesParam` is always a `string` as I see it [here](https://github.com/elastic/kibana/pull/120598/files#diff-8dee193141a2cd84cf80a0f431c2cacdabdc9f0f72c8877028136cb132a94b1aR16)

Reply:
Technically yes, but because this comes from possible user input I wanted to make sure I validate it anyway.